src/components/02-organisms/CategorizedListRoot/Stage/index.js

```diff
@@ -21,8 +21,10 @@ const Stage = props => {
       if (interval) {
         if (props.loopChanges[nodeIndex + 1]) {
           setNodeIndex(nodeIndex + 1);
-        } else {
+        } else if (props.isLoopEnabled) {
           setNodeIndex(0);
+        } else {
+          console.info("Procedure ended.");
```

Comment:
What useful info does this carry in the overall log stream? At least add which procedure and some context, or remove altogether.

Reply:
Stage component is used only with Storybook.